examples/cluster/cluster.yaml

```diff
@@ -4,6 +4,12 @@ kind: Cluster
 metadata:
   name: ${CLUSTER_NAME}
 spec:
+  clusterNetwork:
+    services:
+      cidrBlocks: ["10.96.0.0/12"]
+    pods:
+      cidrBlocks: ["192.168.0.0/16"]
+    serviceDomain: "cluster.local"
```

Comment:
services and serviceDomain can be omitted now.

Reply:
Removed!